src/style/shared/color-picker.less

```diff
@@ -121,7 +121,7 @@
     }
 
     &.color-picker-map__dark .color-picker-map__pointer {
-        border-color: @warm-white;
+        border-color: @white-active;
```

Comment:
Some day it would be great to rename all these variables so that they are color agnostic. For example, instead of `@white-active` it might be `@foreground-active` or something like that.